server/src/internalClusterTest/java/org/opensearch/indices/CacheStatsAPIIndicesRequestCacheIT.java

```diff
@@ -248,44 +287,39 @@ private static Map<String, Object> getNodeCacheStatsXContentMap(Client client, L
         // Can always get the first data node as there's only one in this test suite
         assertEquals(1, nodeStatsResponse.getNodes().size());
         NodeCacheStats ncs = nodeStatsResponse.getNodes().get(0).getNodeCacheStats();
+        return ncs.getStatsByCache(CacheType.INDICES_REQUEST_CACHE);
+    }
 
+    private static Map<String, Object> getStatsXContent(ImmutableCacheStatsHolder statsHolder) throws IOException {
         XContentBuilder builder = XContentFactory.jsonBuilder();
         Map<String, String> paramMap = new HashMap<>();
-        if (aggregationLevels != null && !aggregationLevels.isEmpty()) {
-            paramMap.put("level", String.join(",", aggregationLevels));
-        }
```

Comment:
Why are the aggregationLevels not needed anymore as part of params?

Reply:
I believe during review for the original cache stats API PR, when these original ITs were written, there was a change to how params worked so that levels got passed into the constructor for ImmutableCacheStatsHolder. Before that, we had to also pass them in `params` to ImmutableCacheStatsHolder.toXContent(). But I forgot to remove these lines from the ITs even though they were no longer needed.